pkg/manager/member/utils.go

```diff
@@ -497,3 +500,38 @@ func CreateOrUpdateService(serviceLister corelisters.ServiceLister, serviceContr
 	}
 	return nil
 }
+
+func getStoreByOrdinal(name string, status v1alpha1.TiKVStatus, ordinal int32) *v1alpha1.TiKVStore {
```

Comment:
Should we just update the functions in `tikv_upgrader.go` so that they can also be used in the `pkg/manager/member/tikv_member_manager.go`?